.github/workflows/build-and-release.yaml

```
@@ -0,0 +1,101 @@
+    outputs:
+      MAC_BUILD_HASH: ${{ steps.calc-hash.outputs.MAC_BUILD_HASH }}
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Download MacOS artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: holmes-macos-latest-${{ github.ref_name }}
+    - name: Calculate hash
+      id: calc-hash
+      run: echo "::set-output name=MAC_BUILD_HASH::$(sha256sum holmes-macos-latest-${{ github.ref_name }}.zip | awk '{print $1}')"
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)